### PR TITLE
enh(ui): Add Dialog submitting indicator + update packages

### PR DESCRIPTION
**Pull request by bdauria**

### CODE REVIEW

File: src/Dialog/index.tsx [hunk 1]
```diff
@@ -0,0 +1,66 @@
+import React from 'react';
```
Comment (jdelpierre) [inline src/Dialog/index.tsx]: import * as React from 'react';

File: src/Dialog/index.tsx [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+  onCancel,
+  onConfirm,
+  labelTitle,
+  labelCancel = 'Cancel',
+  labelConfirm = 'Confirm',
+  children,
+  contentWidth,
+  confirmDisabled = false,
+  submitting = false,
+  ...rest
```
Comment (jdelpierre) [inline src/Dialog/index.tsx]: What is "rest" term ?

Comment (bdauria) [reply src/Dialog/index.tsx]: we destructure the props that we don't explicitly use in order to forward them (here to the `MuiDialog`)

Comment (jdelpierre) [reply src/Dialog/index.tsx]: Yes, that's okay, but where did "rest" come from? Cause I couldn't find a match. Is that a reserved term?

Comment (Thebarda) [reply src/Dialog/index.tsx]: Maybe  `...props` is more appropriate ?

**Pull request merged**